src/main/java/org/breedinginsight/services/validators/TraitFileValidatorError.java

```diff
@@ -84,22 +84,27 @@ public ValidationError getMissingScaleCategoriesMsg() {
     }
 
     @Override
-    public ValidationError getBlankScaleCategoryLabelMsg(Integer categoryIndex) {
+    public ValidationError getBadScaleCategory() {
+        return new ValidationError("scale.categories", "Scale categories contain errors", HttpStatus.UNPROCESSABLE_ENTITY);
```

Comment:
I think "scale.categories" needs to change to "Scale Categories"

Reply:
Changed and pushed